shared-libs/environment/src/index.js

```diff
@@ -78,8 +77,13 @@ const getDeployInfo = async () => {
   }
 
   try {
+    // Lazy load couch-request only when needed
+    const request = require('@medic/couch-request');
     const ddoc = await request.get({
       url: `${couchUrl}/_design/${module.exports.ddoc}`,
+      headers: {
+        'user-agent': 'Community Health Toolkit/4.18.0 (test-platform,test-arch)',
```

Comment:
Sorry to nitpick, but can we change this so it doesn't say 4.18.0? It doesn't matter what it is, but would still prefer it to not be inaccurate

```suggestion
        'user-agent': 'Community Health Toolkit',
```